src/main/java/com/minecolonies/coremod/generation/defaults/DefaultResearchProvider.java

```diff
@@ -612,6 +618,63 @@ public DefaultResearchProvider(final DataGenerator generator)
           .addItemCost(Items.DIAMOND, 1)
           .addEffect(ModBuildings.mysticalSite.getBuildingBlock(), 1)
           .addToList(r);
+
+        // Primary Research #6
+        final Research remembrance = new Research(new ResourceLocation(Constants.MOD_ID, "civilian/remembrance"), CIVIL)
+                .setTranslatedName("Remembrance")
+                .setSortOrder(6)
+                .setIcon(ModBlocks.blockHutGraveyard.asItem())
+                .addEffect(ModBuildings.graveyard.getBuildingBlock(), 1)
+                .addBuildingRequirement(ModBuildings.TOWNHALL_ID, 2)
+                .addItemCost(Items.BONE, 8)
+                .setTranslatedSubtitle("Our fallen shall not be forgotten!")
+                .addToList(r);
+
+        final Research resurrectChance1 = new Research(new ResourceLocation(Constants.MOD_ID, "civilian/resurrectchance1"), CIVIL)
+                .setParentResearch(remembrance)
+                .setTranslatedName("Resurrect Change I")
+                .setTranslatedSubtitle("Use the right words")
+                .setSortOrder(1)
+                .setIcon(ModBlocks.blockHutGraveyard.asItem())
+                .addBuildingRequirement(ModBuildings.GRAVEYARD_ID, 3)
+                .addItemCost(Items.GHAST_TEAR, 1)
+                .addEffect(RESURRECT_CHANCE, 1)
+                .addToList(r);
+
+        new Research(new ResourceLocation(Constants.MOD_ID, "civilian/resurrectchance2"), CIVIL)
+                .setParentResearch(resurrectChance1)
+                .setTranslatedName("Resurrect Change II")
```

Comment:
```suggestion
                .setTranslatedName("Resurrection Chance II")
```
Make sure to runData again btw